baybe/utils/interval.py

```diff
@@ -90,15 +120,15 @@ def _(cls, bounds: Iterable):
         """Overloaded implementation for creating an interval of an iterable."""
         return Interval(*bounds)
 
-    def to_tuple(self):
-        """Transfor the interval to a tuple."""
+    def to_tuple(self) -> Tuple[float, float]:
+        """Transform the interval to a tuple."""
         return self.lower, self.upper
 
-    def to_ndarray(self):
+    def to_ndarray(self) -> np.ndarray:
         """Transform the interval to a ndarray."""
```

Comment:
Please add backticks around the `ndarray`.

Reply:
done, also for the tensor. Looking correct in the rendered doc. Resolve if OK 👍🏼

Reply:
Looks good to me. So it does also create the link to the corresponding classes, although they are external references? That is interesting, I did not know that this actually works.